pages/collections/explore/index.vue

```diff
@@ -0,0 +1,356 @@
+    {
+      hid: 'description',
+      name: 'description',
+      content: removeTags(page.value.summary),
+    }
+  ],
+})
+
+// COMPUTED
+const parsedCollectionList = computed(() => {
+  return collections.value.map((obj) => {
+    return {
+      ...obj,
+      to: obj.externalResourceUrl
+        ? obj.externalResourceUrl
+        : `/${obj.uri}`,
+      image: _get(obj, 'heroImage[0]image[0]', null),
+      category: obj.category.join(', '),
+      title: _get(obj, 'title', ''),
+      text: _get(obj, 'text', ''),
+    }
+  })
+})
+
+const parsedAssociatedTopics = computed(() => {
+  return page.value.associatedTopics.map((obj) => {
+    return {
+      ...obj,
+      to: obj.externalResourceUrl
+        ? obj.externalResourceUrl
+        : `/${obj.uri}`,
+    }
+  })
+})
+
+// ELASTIC SEARCH FUNCTIONALITY
+const hits = ref([])
+const noResultsFound = ref(false)
+const searchFilters = ref([])
+const searchGenericQuery = ref({
+  queryText: route.query.q || '',
+  queryFilters:
+    (route.query.filters &&
+      JSON.parse(route.query.filters)) ||
+    {},
+})
+
+// This watcher is called when router pushes updates the query params
+watch(
+  () => route.query,
+  (newVal, oldVal) => {
+    console.log('ES newVal, oldVal', newVal, oldVal)
+    searchGenericQuery.value.queryText = route.query.q || ''
+    searchGenericQuery.value.queryFilters = (route.query.filters && JSON.parse(route.query.filters)) || {}
+    searchES()
+  }, { deep: true, immediate: true }
+)
+
+// Elastic Search Function
+async function searchES() {
+  // collections.value = []
+  // hits.value = []
+  if (
+    (route.query.q && route.query.q !== '') ||
+    (route.query.filters &&
+      queryFilterHasValues(
+        route.query.filters,
+        config.exploreCollection.filters
+      ))
+  ) {
+    if (!page.title) {
+      const data = await $graphql.default.request(
+        COLLECTIONS_EXPLORE_LIST
+      )
+      page.title = _get(data, 'entry.title', '')
+      page.text = _get(data, 'entry.text', '')
+    }
+
+    console.log('Search ES HITS query,', route.query.q)
+    const queryText = route.query.q || '*'
+    const results = await $dataApi.keywordSearchWithFilters(
+      queryText,
+      config.exploreCollection.searchFields,
+      'sectionHandle:collection',
+      (route.query.filters &&
+        JSON.parse(route.query.filters)) ||
+      {},
+      config.exploreCollection.sortField,
+      config.exploreCollection.orderBy,
+      config.exploreCollection.resultFields,
+      config.exploreCollection.filters
+    )
+    console.log('getsearchdata method:' + JSON.stringify(results))
+    collections.value = []
+    hits.value = []
+    if (results && results.hits && results.hits.total.value > 0) {
+      hits.value = results.hits.hits
+      collections.value = []
+      noResultsFound.value = false
+    } else {
+      hits.value = []
+      collections.value = []
+      noResultsFound.value = true
+    }
+    searchGenericQuery.value = {
```

Comment:
Remove this line 
` searchGenericQuery.value = {
      queryText: route.query.q || '',
      queryFilters:
        (route.query.filters &&
          JSON.parse(route.query.filters)) ||
        {},
    }`